src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/DoNotInitializeUnnecessarilyTests.cs

```diff
@@ -311,6 +312,101 @@ public class Test
                     .WithLocation(3));
         }
 
+        [Fact, WorkItem(5750, "https://github.com/dotnet/roslyn-analyzers/issues/5750")]
+        public async Task ParameterlessValueTypeCtor()
+        {
+            await new VerifyCS.Test
+            {
+                TestCode = @"
+using System;
+
+struct S
+{
+    public static readonly S Value = new();
+    public static readonly TimeSpan Time = new();
```

Comment:
With my initial change we were reporting here but now that we exclude all instance members of struct we no longer report. Let me know if you would prefer to have a few hardcoded types for which we know we can still "simplify" initialization.